pkg/operation/seed/seed.go

```diff
@@ -673,6 +678,82 @@ func BootstrapCluster(ctx context.Context, k8sGardenClient, k8sSeedClient kubern
 		}
 	}
 
+	schedulerImage, err := imageVector.FindImage(
+		common.KubeSchedulerImageName,
+		imagevector.TargetVersion(*seed.Info.Status.KubernetesVersion),
+	)
+	if err != nil {
+		return err
+	}
+
+	seedVersion, err := semver.NewVersion(*seed.Info.Status.KubernetesVersion)
+	if err != nil {
+		return err
+	}
+
+	scheduler, err := scheduler.New(
+		k8sSeedClient.DirectClient(),
+		"gardener-kube-scheduler",
+		schedulerImage.String(),
+		seedVersion,
+		&schedulerconfigv1alpha2.KubeSchedulerConfiguration{
+			Profiles: []schedulerconfigv1alpha2.KubeSchedulerProfile{{
+				SchedulerName: pointer.StringPtr(scheduler.GardenerShootControlPlaneSchedulerName),
+				Plugins: &schedulerconfigv1alpha2.Plugins{
+					Score: &schedulerconfigv1alpha2.PluginSet{
+						Disabled: []schedulerconfigv1alpha2.Plugin{
+							{Name: "NodeResourcesLeastAllocated"},
+							{Name: "NodeResourcesBalancedAllocation"},
+						},
+						Enabled: []schedulerconfigv1alpha2.Plugin{
+							{Name: "NodeResourcesMostAllocated"},
+						},
+					},
+				},
+			}},
+		},
+		&admissionregistrationv1beta1.WebhookClientConfig{
+			Service: &admissionregistrationv1beta1.ServiceReference{
+				Name:      "gardener-seed-admission-controller",
+				Namespace: v1beta1constants.GardenNamespace,
+				Path:      pointer.StringPtr(scheduler.GardenerShootControlPlaneSchedulerWebhookPath),
+			},
+			// TODO(mvladev) this cert is hard-coded in the admission chart.
+			// fix it in another PR.
+			CABundle: []byte(`-----BEGIN CERTIFICATE-----
+MIIC+jCCAeKgAwIBAgIUTp3XvhrWOVM8ZGe86YoXMV/UJ7AwDQYJKoZIhvcNAQEL
+BQAwFTETMBEGA1UEAxMKa3ViZXJuZXRlczAeFw0xOTAyMjcxNTM0MDBaFw0yNDAy
+MjYxNTM0MDBaMBUxEzARBgNVBAMTCmt1YmVybmV0ZXMwggEiMA0GCSqGSIb3DQEB
+AQUAA4IBDwAwggEKAoIBAQCyi0QGOcv2bTf3N8OLN97RwsgH6QAr8wSpAOrttBJg
+FnfnU2T1RHgxm7qd190WL8DChv0dZf76d6eSQ4ZrjjyArTzufb4DtPwg+VWq7XvF
+BNyn+2hf4SySkwd6k7XLhUTRx048IbByC4v+FEvmoLAwrc0d0G14ec6snD+7jO7e
+kykQ/NgAOL7P6kDs9z6+bOfgF0nGN+bmeWQqJejR0t+OyQDCx5/FMtUfEVR5QX80
+aeefgp3JFZb6fAw9KhLtdRV3FP0tz6hS+e4Sg0mwAAOqijZsV87kP5GYzjtcfA12
+lDYl/nb1GtVvvkQD49VnV7mDnl6mG3LCMNCNH6WlZNv3AgMBAAGjQjBAMA4GA1Ud
+DwEB/wQEAwIBBjAPBgNVHRMBAf8EBTADAQH/MB0GA1UdDgQWBBSFA3LvJM21d8qs
+ZVVCe6RrTT9wiTANBgkqhkiG9w0BAQsFAAOCAQEAns/EJ3yKsjtISoteQ714r2Um
+BMPyUYTTdRHD8LZMd3RykvsacF2l2y88Nz6wJcAuoUj1h8aBDP5oXVtNmFT9jybS
+TXrRvWi+aeYdb556nEA5/a94e+cb+Ck3qy/1xgQoS457AZQODisDZNBYWkAFs2Lc
+ucpcAtXJtIthVm7FjoAHYcsrY04yAiYEJLD02TjUDXg4iGOGMkVHdmhawBDBF3Aj
+esfcqFwji6JyAKFRACPowykQONFwUSom89uYESSCJFvNCk9MJmjJ2PzDUt6CypR4
+epFdd1fXLwuwn7fvPMmJqD3HtLalX1AZmPk+BI8ezfAiVcVqnTJQMXlYPpYe9A==
+-----END CERTIFICATE-----`),
+		},
+	)
+	if err != nil {
+		return err
+	}
+
+	if gardenletfeatures.FeatureGate.Enabled(features.SeedKubeScheduler) {
+		if err := scheduler.Deploy(ctx); err != nil {
+			return err
+		}
+	} else {
+		if err := scheduler.Destroy(ctx); err != nil {
+			return err
+		}
+	}
```

Comment:
Can you move this into https://github.com/gardener/gardener/blob/master/pkg/operation/seed/seed.go#L801-L827?

Reply:
Done